formatter/pretty.go

```diff
@@ -87,11 +88,12 @@ func (f *Formatter) printIssueWithSource(issue *tflint.Issue, sources map[string
 }
 
 func (f *Formatter) printErrors(err *tflint.Error, sources map[string][]byte) {
-	if diags, ok := err.Cause.(hcl.Diagnostics); ok {
+	var diags *hcl.Diagnostics
+	if errors.As(err.Cause, &diags) {
```

Comment:
This should be `errors.As(err, &diags)` which may require amending the implementation of `tflint.Error` to play nicely with error wrapping. 

The `Cause` field in a `tflint.Error` is its own form of error chaining. `tflint.Error` should implement `As` to express the fact that `Cause` is considered part of the error chain.

> An error type might provide an As method so it can be treated as if it were a different error type.